src/Cli/dotnet/Commands/Workload/InstallStateContents.cs

```diff
@@ -40,7 +40,7 @@ public static InstallStateContents FromPath(string path)
 
     public override string ToString()
     {
-        return JsonSerializer.Serialize<InstallStateContents>(this, s_options);
+        return JsonSerializer.Serialize(this, s_options);
```

Comment:
I prefer having the type here. Not a super strong preference, though

Reply:
Since it automatically removed it, I'll let stay removed. It is just likely to try to remove it again when I run another tool that would hit this file.